test/integration/fedora-27/multiple.test

```diff
@@ -13,11 +13,11 @@ die() {
 
 ko_to_test() {
 	tmp=${1%.ko}-LOADED.test
-	echo ${tmp#kpatch-}
+	echo ${tmp#test-}
 }
 
 # make sure any modules added here are disjoint
-declare -a modules=(kpatch-cmdline-string.ko kpatch-meminfo-string.ko)
+declare -a modules=(test-data-new.ko test-shadow-newpid.ko)
```

Comment:
Maybe. Are the modules in this list just the ones that have a corresponding *.test? I am also not sure we'll be able to make sure the modules are disjoint, as the comment above this line suggests, if we build this list dynamically.

Reply:
Yes, I think it should just be all modules that have a corresponding *.test.  If they're not disjoint then the test will fail (due to insmod failing) and we can cross that bridge if/when we get there :-)

Reply:
Aaaand we are already there. On ubuntu/centos we have both data-new and meminfo-string patches with corresponding tests that both patch meminfo_proc_show().

Reply:
I guess have to hard-code the tests for ubuntu/centos then.

Reply:
I was wondering whether we can just delete one of these tests instead. They are pretty similar so we probably won't lose much coverage. This will also be a problem on fedora once we reenable meminfo-cmdline-rebuild-SLOW-LOADED.test

Reply:
Or maybe we can keep the dynamic test detection, but then add a hard-coded blacklist.  That way we don't have to delete tests, and we'll know the multiple.test blacklist needs to be updated when the tests fail due to a conflict.